frontend/src/App.js

```diff
@@ -71,31 +71,69 @@ class App extends Component {
     this.uniqueKey = 0;
 
     this.exampleQueries = [
+      {
+        name: 'Files named main.go',
+        sql: `/* Files named main.go in HEAD */
+SELECT t.repository_id, t.tree_entry_name,
+       language(t.tree_entry_name, b.blob_content) AS lang, b.blob_content,
+       uast(b.blob_content, language(t.tree_entry_name, b.blob_content))
+FROM   tree_entries AS t
+       JOIN blobs b ON tree_entries.blob_hash = blobs.blob_hash
+       JOIN commit_trees ON tree_entries.tree_hash = commit_trees.tree_hash
+       JOIN refs ON commit_trees.commit_hash = refs.commit_hash
+WHERE  ref_name = 'HEAD'
+       AND tree_entries.tree_entry_name = 'main.go'`
+      },
+      {
+        name: 'Last commit for each repository',
+        sql: `/* Last commit for each repository */
+SELECT r.repository_id,commit_author_name,commit_author_when,commit_message
```

Comment:
should we mind the spaces in the same style? Right now I see some examples have spaces but others don't